setup.py

```diff
@@ -37,7 +37,7 @@
     include_package_data=True,
     install_requires=[
         "python-dotenv",
-        "Click==7.1",
+        "Click>=7.1",
```

Comment:
Assuming Click follows semantic versioning, should we specify >=7.1,<8? This would prevent us from automatically pulling in a Click that the Click developers feel is not backwards compatible with 7.x.